pnpm-lock.yaml

```diff
@@ -1,4 +1,8 @@
-lockfileVersion: '6.0'
```

Comment:
Considering I made no changes to dependencies etc. I am not sure what to make of these changes to the lock file. Happy to revert changes to this file if they are not necessary or erroneous.